framework/src/meshgenerators/SideSetsBetweenSubdomainsGenerator.C

```diff
@@ -20,67 +20,44 @@ registerMooseObject("MooseApp", SideSetsBetweenSubdomainsGenerator);
 InputParameters
 SideSetsBetweenSubdomainsGenerator::validParams()
 {
-  InputParameters params = MeshGenerator::validParams();
+  InputParameters params = SideSetsGeneratorBase::validParams();
 
-  params.addRequiredParam<MeshGeneratorName>("input", "The mesh we want to modify");
-  params.addParam<std::vector<SubdomainName>>(
-      "primary_block", "The primary set of blocks for which to draw a sideset between");
   params.addDeprecatedParam<std::vector<SubdomainName>>(
-      "master_block",
+      "primary_block",
       "The primary set of blocks for which to draw a sideset between",
-      "The 'master_block' param is deprecated and will be removed on January 1, 2021. Please use "
-      "the 'primary_block' parameter instead.");
-  params.addRequiredParam<std::vector<SubdomainName>>(
-      "paired_block", "The paired set of blocks for which to draw a sideset between");
-  params.addRequiredParam<std::vector<BoundaryName>>("new_boundary",
-                                                     "The name of the boundary to create");
+      "Deprecated, use 'included_subdomains'");
+  params.deprecateParam("primary_block", "included_subdomains", "4/01/25");
```

Comment:
Deprecation of `primary_block` happens here. See civet job https://civet.inl.gov/job/2115751/

Reply:
Would you mind reviewing everything so far? That way, I don't have to switch between tasks as much.